lib/debootstrap.sh

```diff
@@ -444,8 +444,9 @@ prepare_partitions()
 	if [[ $BUILD_DESKTOP == yes ]]; then local node_number=4096; else local node_number=1024; fi
 	if [[ $HOSTRELEASE =~ buster|bullseye|focal|jammy|sid ]]; then
 		mkopts[ext4]="-q -m 2 -O ^64bit,^metadata_csum -N $((128*${node_number}))"
+		display_alert "ext4 mkfs options calculated as" "${mkopts[ext4]}" "info"
 	fi
-	# mkopts[fat] is empty
+	mkopts[fat]=' -F 16 '
```

Comment:
I have a question.
Why 16 bits?

Reply:
boot partition size is well unde 4 GB limit of fat16. Both 16 and 32 should work with u-boot. Just chose most conservative. But if for some reason you prefer fat32, fine with me. Have not tested fat32 to see whether still the system type is recognized as vfat when rsync is used to fill the file system. It should. I added a log output there to confirm that some form of vfat is detected.